shell/platform/darwin/ios/framework/Headers/FlutterTexture.h

```diff
@@ -15,6 +15,14 @@ NS_ASSUME_NONNULL_BEGIN
 FLUTTER_EXPORT
 @protocol FlutterTexture <NSObject>
 - (CVPixelBufferRef _Nullable)copyPixelBuffer;
+
+/**
+ * Called when the texture is unregistered.
+ *
+ * Called on the GPU thread.
+ */
+@optional
+- (void)onUnregistered;
```

Comment:
Ditto about the `onUnregistered` begin too ambiguous. In Objective-C the idiom is to also pass the object as an argument. So something like:

`-(void) onTextureUnregistered:(FlutterTexture*) texture;`